projects/go-lib/src/lib/components/go-search/go-search.component.scss

```diff
@@ -14,7 +14,7 @@
   @include transition(all);
 
   background: $theme-light-bg;
-  border: 1px solid $theme-light-border;
```

Comment:
Are we sure about this one? This change makes the border on the search _very_ light. 

![Screen Shot 2020-10-29 at 4 46 35 PM](https://user-images.githubusercontent.com/11966331/97630685-500bab00-1a06-11eb-9513-422f600e08c6.png)

@kpmurphh can you confirm this for us?

Reply:
@grahamhency @Ashygoyal I can't tell what color it is from this code or this image, but it should match the other inputs border color, which is #D3D3D3.

Reply:
@kpmurphh As per the DSM [guidelines](https://tangoe.invisionapp.com/dsm/tangoe/tangoe-ds/folder/colors/5c9435edae717d86ac2b0c6d), the search border color should be `#F0F0F0`
![image](https://user-images.githubusercontent.com/3825309/98004428-edefe300-1e15-11eb-928a-c768c20e4ee1.png)

I hope I am referring to the updated DSM URL. Can you please confirm.
For this PR, I'll make the search border as `#D3D3D3`.

Reply:
@grahamhency I have updated this PR, ready for one last review!
Thanks!